### PR TITLE
__props__ to basic.py

### PR DESCRIPTION
**Pull request by harlouci**
All __props__  checked in the folder theano/typed_list

### CODE REVIEW

**Comment by nouiz** [comment]
ok to merge when travis pass.

File: theano/typed_list/basic.py [hunk 1]
```diff
@@ -116,6 +111,8 @@ def c_code_cache_version(self):
 
 class Append(Op):
     # See doc in instance of this Op after the class definition.
+    __props__ = ("inplace")
```
Comment (nouiz) [inline theano/typed_list/basic.py]: need comma as elsewhere in the file.

**Comment by nouiz** [comment]
travis fail.

**Comment by harlouci** [comment]
Idem, no errors when I execute

flake8 --ignore E501 theano/tensor/extra_ops.py

**Pull request merged**